### PR TITLE
Focus on selected node

### PR DESCRIPTION
**Pull request by scottmitchell**
Now selects and focuses on the node that is selected in the TuneUp table.

FocusOnSelected

@mjkkirschner @QilongTang

### CODE REVIEW

File: TuneUp/TuneUpWindow.xaml.cs [hunk 1]
```diff
@@ -23,46 +23,49 @@ public partial class TuneUpWindow : Window
...
+                selectedNodes.Add((item as ProfiledNodeViewModel).NodeModel);
             }
-            foreach (var row in e.RemovedItems)
+
+            if (selectedNodes.Count() > 0)
             {
-                (row as ProfiledNodeViewModel).NodeModel.Deselect();
+                // Select
+                var command = new DynamoModel.SelectModelCommand(selectedNodes.Select(nm => nm.GUID), ModifierKeys.None);
+                viewLoadedParams.CommandExecutive.ExecuteCommand(command, Guid.NewGuid().ToString(), "TuneUp");
```
Comment (mjkkirschner) [inline TuneUp/TuneUpWindow.xaml.cs]: this should be the guid of the extension, not a new guid.

File: TuneUp/TuneUpWindow.xaml.cs [hunk 2]
```diff
@@ -23,46 +23,49 @@ public partial class TuneUpWindow : Window
...
+
+                // Focus on selected
+                var ce = viewLoadedParams.GetType().GetProperty("ViewModelCommandExecutive");
```
Comment (mjkkirschner) [inline TuneUp/TuneUpWindow.xaml.cs]: can you rip this string out to a const and add a comment here saying why you are doing this using reflection for API compatibility.

**Comment by mjkkirschner** [comment]
@scottmitchell I think this looks good, have you thought about how you might write an automated test?

**Comment by scottmitchell** [comment]
@mjkkirschner @QilongTang @ColinDayOrg This PR is ready for a final review

**Comment by scottmitchell** [comment]
@mjkkirschner I haven't thought about testing much yet, because we haven't yet gotten to the stuff we want to test. We are going to want to test profiling data related to delta-compute, force re-execute, etc., which is begins with the next task: https://github.com/DynamoDS/TuneUp/issues/2. How about we add our first test there?

File: TuneUp/TuneUpWindow.xaml [hunk 3]
```diff
@@ -78,14 +78,14 @@
             FontSize="11"
             VerticalAlignment="Center"
             SelectionUnit="FullRow"
-            SelectionMode="Extended"
+            SelectionMode="Single"
```
Comment (QilongTang) [inline TuneUp/TuneUpWindow.xaml]: @ScottAMitchell Is this the difference between multiple selection and single selection?

File: TuneUp/TuneUpWindow.xaml.cs [hunk 4]
```diff
@@ -23,46 +23,44 @@ public partial class TuneUpWindow : Window
 
         ViewLoadedParams viewLoadedParams;
 
+        ICommandExecutive commandExecutive;
+
+        ViewModelCommandExecutive viewModelCommandExecutive;
+
+        string uniqueId;
```
Comment (QilongTang) [inline TuneUp/TuneUpWindow.xaml.cs]: Can you put some comments on this property and usage?

**Comment by QilongTang** [comment]
Other than the two comments, LGTM

File: TuneUp/TuneUpWindow.xaml (hunk 3 above)
Comment (scottmitchell) [reply TuneUp/TuneUpWindow.xaml]: Yes, I limited it to single selection to make the `centerViewOnElement` ux consistent

**Pull request merged**